common/views/components/PrototypeSearchFilters/PrototypeSearchFiltersDesktop.tsx

```diff
@@ -355,14 +357,21 @@ const SearchFiltersDesktop = ({
                 <NextLink
                   passHref
                   {...imagesLink(
-                    { ...worksRouteProps, page: 1, color: null },
+                    {
+                      ...worksRouteProps,
+                      page: 1,
+                      color: null,
+                      locationsLicense: null,
+                    },
                     'cancel_filter/images_color'
                   )}
                 >
                   <a>
                     <CancelFilter>
-                      Colour
-                      <ColorSwatch color={`#${imagesColor}`} />
+                      <>
```

Comment:
is the extra fragment necessary?

Reply:
Yip, due to the type required by `CancelFilter`.

~I have changed the type to React.ReactNode, as I would have to have a complex type of something like `JSX.Element | JSX.Element[] | string | string[]` - which isn't the children type.~

I have now just typed the `CancelFilter` as a `FunctionalComponent` which has `children` as a prop automagically.